default/stringtables/en.txt

```diff
@@ -339,11 +339,14 @@ PROJECT_BT_STOCKPILE_SHORT_DESC
 Transfers PP into the imperial stockpile
 
 PROJECT_BT_STOCKPILE_DESC
-'''Transfers PP at the planet where this project is enqueued to the imperial stockpile. Anything enqueued below the transfer project at a supply-connected planet will not be allocated PP.
+'''Transfers PP at the planet where this project is enqueued to the imperial stockpile. One can specify the amount and repetition of the transfer. PP spent on the transfer are added to the imperial stockpile.
```

Comment:
"One can specify the amount and repetition of the transfer" -> "The amount to transfer and for how many turns the transfer should occur can be specified"

Reply:
I wouldnt want to use "turns" here. If the stockpile project is not fully funded you dont specify a number of turns.

Reply:
OK, but the stylistic changes remain.